package.json

```diff
@@ -3,7 +3,7 @@
   "scripts": {
     "clean": "rimraf output && rimraf .pulp-cache",
     "build": "eslint src && pulp build -- --censor-lib --strict",
-    "test": "pulp test"
+    "test": "pulp test -- --censor-lib --strict"
```

Comment:
I don't think this is necessary, but not an issue either.

Reply:
I think it might help if all of our files were the same as it means we can more easily update them via scripts later.